lib/babel.js

```diff
@@ -16,6 +16,11 @@ if (debug.enabled) {
 // Bind globals first, before anything has a chance to interfere.
 var globals = require('./globals');
 
+var resolveCwd = require('resolve-cwd');
+(opts.require || []).forEach(function (moduleId) {
```

Comment:
Nope. In the API tests, `new Api()` is called without the second parameter, which means it ends up being an empty Object, without a "require" property. Should I change the fallback default value in api.js to include an empty "require" array? e.g. `this.options = options || { require: [] };` ?